include/taskolib/Sequence.h

```diff
@@ -490,6 +521,7 @@ class Sequence
     std::string indentation_error_;
 
     std::string label_;
+    std::string step_setup_;
```

Comment:
Maybe this should be called `step_setup_script_` to differentiat that from `Context.lua_step_setup`.